packages/calcite-components/src/utils/dynamicClasses.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { getDimension } from "./dynamicClasses";
+
+describe("getDimension", () => {
+  const types = ["width", "height"] as const;
+  const sizes = ["s", "m", "l", ""] as const;
+  const scales = ["s", "m", "l", ""] as const;
```

Comment:
You'll need to use `undefined` since `""` isn't a valid value for these properties.